src/erf.jl

```diff
@@ -413,7 +415,55 @@ function erfcinv(y::Float32)
     end
 end
 
-erfcinv(x::Integer) = erfcinv(float(x))
+function erfinv(y::BigFloat)
+    xfloat = erfinv(Float64(y))
+    sqrtπ = sqrt(big(pi))
+    if isfinite(xfloat)
+        x = BigFloat(xfloat)
+    else
+        # Float64 overflowed, use asymptotic estimate instead
+        # from erfc(x) ≈ exp(-x²)/x√π ≈ y  ⟹  -log(yπ) ≈ x² + log(x) ≈ x²
+        x = copysign(sqrt(-log((1-abs(y))*sqrtπ)), y)
+    end
+    isfinite(x) || return x
```

Comment:
should this be moved into the end of the else block? 

`isfinite(xfloat)` will already ensure `x` in L422 is finite, so this looks like it's checking `isfinite(x)` twice in that case. Not a big deal, but thought I'd mention.

Reply:
You need both checks because xfloat might overflow even if the result is representable as bigfloat

Reply:
I'm not sure if I was clear (I might have worded tings confusingly). I was hoping to say:
why don't you move `isfinite(x) || return x` right after L426, instead of after the `if...else...end`  block.

Reply:
Yes, that would work.

Reply:
Fixed in cdd323d00a4fc4b6dc93c41cf24406f0b7208e5d